README.md

```diff
@@ -23,7 +23,7 @@ with the chip on your system and start the emulated device.
 1. AS A NONROOT USER: In the `rust/` directory, run `./build.sh`.
    This step will take a long time.
 2. From the root of the project, run `yarn install`
-3. Run `npm run android` to compile and install the app on an
+3. Run `yarn run android` to compile and install the app on an
```

Comment:
wait though .. is it not just `yarn android` then?

Reply:
ah, apparently the `run` is optional - `yarn android` or `yarn run android`; so, it's fine ...